client/src/App.jsx

```diff
@@ -38,55 +43,51 @@ function App() {
 
   return (
     <ThemeProvider theme={siteTheme}>
-      <Box className="App" sx={{ bgcolor: "primary.light" }}>
-        <BrowserRouter>
-          {/* Landing Page */}
-          <Route exact path="/">
-            <Landing />
-          </Route>
+      <UserContextProvider value={(registeredUser, isLoggedIn)}>
```

Comment:
You don't need to pass values when rendering the Provider -- you should be adding these values in `client/src/components/UserContext.jsx`.

Also, the `value={(registeredUser, isLoggedIn)}` syntax is off: if you wrap variables in (), then javascript thinks you're trying to evaluate an expression. So what's getting passed here is likely something like `value={undefined}` or `value={null}`.